src/App.tsx

```diff
@@ -1,10 +1,66 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+// #region imports
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import { getTodos, USER_ID } from './api/todos';
+import { Todo } from './types/Todo';
+import classNames from 'classnames';
+import { Header } from './components/Header';
+import { TodoList } from './components/TodoList';
+import { Footer } from './components/Footer';
+// #endregion
 
 export const App: React.FC = () => {
+  // #region useState
+
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [allTodos, setAllTodos] = useState<Todo[]>([]);
+  const [selectedTodos, setSelectedTodos] = useState<number[]>([]);
+  const [selectedLink, setSelectedLink] = useState('All');
```

Comment:
```suggestion
  const [selectedLink, setSelectedLink] = useState('All');
```
Consider using an enum here